src/test/java/HandleGetRequestTest.java

```diff
@@ -2,9 +2,9 @@
 import java.util.*;
 import java.io.IOException;
 
-public class HttpHandlerSelectorTest extends junit.framework.TestCase {
+public class HandleGetRequestTest extends junit.framework.TestCase {
 
-  HttpHandlerSelector handler;
+  HandleGetRequest getRequest;
```

Comment:
No. What's confusing is something like line 25 and 26.

Line 25 creates a new HttpRequest that's really a GET request, and then line 26 uses something that's called `getRequest` do generate a response.

I think what's contributing to the confusion are class names that are actually verbs (i.e. `HandleGetRequest`).

Typically, classes are nouns and methods/functions are verbs.